### PR TITLE
Create assignment.rb

### PR DESCRIPTION
**Pull request by Kennethjk3**
Ability to request and allow the input of a users name, and based on he/she's legal values, output a description of what Star Wars character he/she would most associate.

### CODE REVIEW

**Comment by Kennethjk3** [comment]
Confirmed...

File: assignment.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+print "Welcome to the Star Wars Character test\n Please Enter your name: "
+name = gets.chomp
+    puts "Hello #{name}\n"
```
Comment (rpearce) [inline assignment.rb:3]: In Ruby, tabbing/spaces are used to imply that something is "nested" within something else. However, I don't think that's the case here.

File: assignment.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+print "Lets find out who you would be in the Star Wars Universe.\n Go over the statements and type in the number that best\n corresponds with your personality.""\n"
+puts     "1 = I am law abiding!\n
```
Comment (rpearce) [inline assignment.rb:5]: you can remove the spacing here

File: assignment.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    if legal == "1"
+        puts "The Empire thanks you for your aquiescence, citizen #{name}"
+    elsif legal == "2"
+        puts "#{name}, aren't you a little short for a Storm Trooper?"
+    elsif legal == "3"
+        puts "Hey, #{name}! Who's scruffy-lookin'?"
+    elsif legal == "4"
+        puts "#{name}, I loathe bounty hunters. We don't need their scum..."
+    else
+        puts "#{name}, Meesa Jar Jar Binks!"
```
Comment (rpearce) [inline assignment.rb:21]: this whole `if` block should be moved back all the way to the left. but it works!

**Comment by rpearce** [comment]
Apart from the spacing style changes, :+1:. Close and merge this PR at your convenience.

**Pull request merged**